### PR TITLE
Improve README for first-time users

### PR DESCRIPTION
**Pull request by abias**
I tried to slightly improved the README for first-time users. The changes are not perfect, but I tried to clear up the questions I had when working with the tool for the first time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,25 +14,37 @@ This repository contains Docker configuration aimed at Moodle developers and tes
...
 
-## Example usage
+## Quick start
 
 ```bash
-# Set up path to code and choose a db server (pgsql/mssql/oracle/mysql)
+# Set up path to code
 export MOODLE_DOCKER_WWWROOT=/path/to/moodle/code
-export MOODLE_DOCKER_DB=mssql
+# Choose a db server
```
Comment (scara) [inline README.md]: Please, keep its options inline in the comment: it's useful to cut&paste one among them to quickly change the DB server.

TIA,
Matteo

Comment (abias) [reply README.md]: I basically agree with you. But then this list must be maintained. The options table below says "pgsql, mariadb, mysql, mssql, oracle", the comment which I removed said "pgsql/mssql/oracle/mysql".

Comment (scara) [reply README.md]: I agree but the other option is to add a proper check in the shell command: IMHO it's simplier to maintain the list just in the README. Guessing no new DB type will be supported at least in middle term.

Comment (abias) [reply README.md]: OK, I changed this and updated the PR

**Comment by danpoltawski** [comment]
Love this kind of contribution - the first questions from a newcomer are the most important.

Thanks a lot @abias! And thanks @scara for the helpful review comments too!

**Pull request merged**